health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/AddDeliverycontext.js

```
@@ -560,9 +605,9 @@
       )}
     </>
   );
-};
+});
 
-const AddDeliveryRuleWrapper = ({}) => {
+const AddDeliveryRuleWrapper = React.memo(({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)